test/e2e/multihoming.go

```diff
@@ -899,6 +899,85 @@ var _ = Describe("Multi Homing", func() {
 					)
 				})
 			})
+
+			Context("with a trunked configuration", func() {
+				const vlanID = 20
+				BeforeEach(func() {
+					nodes = ovsPods(cs)
```

Comment:
NOTE: this function "ovsPods" assumes that ovs is being ran as a pod, which is a problem for clusters that have ovs as part od systemctl in the node.